firebase-vertexai/src/main/kotlin/com/google/firebase/vertexai/type/GenerationConfig.kt

```diff
@@ -69,6 +69,9 @@ import kotlinx.serialization.Serializable
  * @property responseSchema Output schema of the generated candidate text. If set, a compatible
  * [responseMimeType] must also be set.
  *
+ * @property responseModalities Specifies the format of the data in which the server responds to
+ * requests
```

Comment:
@rlazo Should we also add a note here to make sure your model supports the modalities you specified, or is it just sort of implied?

```suggestion
 * @property responseModalities The format of data in which the model should respond with.
```